src/com/sap/piper/PiperGoUtils.groovy

```diff
@@ -65,4 +68,39 @@ class PiperGoUtils implements Serializable {
         }
         return false
     }
+
+    /*
+     * The returned string can be used directly in the command line for retrieving the configuration via go
+     */
+    public String prepareConfigurations(List configFiles, String configCacheFolder) {
```

Comment:
not sure if we should start refactoring the groovy parts with this step.

From my POV we should rather look into a consolidated way of optimization the groovy parts.

With these changes we have two different styles in the groovy parts of the steps.
Why not re-using the style of the other steps here?

Reply:
>Why not re-using the style of the other steps here?

I can also re-use some other ways how to do this. Can you point to the other way?

Reply:
In order to streamline this I started the following:
https://github.com/SAP/jenkins-library/pull/1254

Goal would be to include all use-cases in this central functionality.
Checkmarx and KubernetesDeploy is currently covered which should cater for the majority of cases already ...

Reply:
#1254 is merged and can be used ...